src/patternfly/components/Table/table.scss

```diff
@@ -78,8 +79,6 @@
   --#{$table}--cell--hidden-visible--Display: table-cell;
 
   // * Table toggle
-  --#{$table}__toggle--PaddingBlockStart: var(--pf-t--global--spacer--sm);
-  --#{$table}__toggle--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
```

Comment:
Similar issue with this one - it's still used here https://github.com/patternfly/patternfly/blob/4ee99f15fbb5760024916e185cda063f19cdda6f/src/patternfly/components/Table/table.scss#L741

And then redefined here https://github.com/patternfly/patternfly/blob/4ee99f15fbb5760024916e185cda063f19cdda6f/src/patternfly/components/Table/table.scss#L1057